lib/phoenix_test/playwright.ex

```diff
@@ -560,6 +560,40 @@ defmodule PhoenixTest.Playwright do
     conn
   end
 
+  @drag_and_drop_opts_schema [
+    target: [
+      type_spec: quote(do: selector()),
+      type_doc: "`t:selector/0`",
+      required: true,
+      doc: "The target selector."
+    ]
+  ]
+
+  @doc """
+  Drag and drop an element to a target element.
+
+  ## Options
+  #{NimbleOptions.docs(@drag_and_drop_opts_schema)}
+
+  ## Examples
+      |> drag_and_drop("#source", target: "#target")
+      |> drag_and_drop(Selector.text("Draggable"), target: Selector.text("Target"))
+  """
+  @spec drag_and_drop(t(), selector(), [
+          unquote(NimbleOptions.option_typespec(@drag_and_drop_opts_schema))
+        ]) :: t()
+  def drag_and_drop(conn, selector, opts) do
```

Comment:
```suggestion
  def drag_to(conn, selector, opts) do
```

Unsure. [playwright JS](https://playwright.dev/docs/input#drag-and-drop) uses `selector.dragTo()`.
But
1. Internally, playwright names it `dragAndDrop`
2. phoenix_test_playwright API is not selector based.
3. We can/should use a named `target` option, so this does not read fluently:

```ex
|> drag_to("#source", target: "#target")
```

Alternatives

```ex
|> drag_to("#source", "#target")
|> drag(from: "#source", to: "#target")
```

Reply:
I like the API and extensibility idea.
What I don't like is that it promises too much and moves future complexity into `phoenix_test_playwright`.
Playwright doesn't support a drag-drop API at that level, and (while it would be great) I don't plan on providing an API that is more high-level than either `phoenix_test` or `playwright`.

That way, there is less you have to learn about `phoenix_test_playwright`.
If you know `phoenix_test`, you can use `phoenix_test_playwright` right away. If you need advanced features, it should be sufficient to know `playwright` (or look at the playwright docs).

Does that make sense?

Reply:
That being said I agree `|> drag(source, to: target)` would be both close to playwright JS `dragTo` and read well with elixir Pipe syntax.